metricbeat/module/vsphere/host/data_host_system.go

```diff
@@ -0,0 +1,129 @@
+package host
```

Comment:
It looks like this file is only used for testing. If so it should be named with the `_test.go` suffix so that it is only compiled and used in tests. But I would probably just move the `HostSystemTest` definition into the `data_test.go`.